src/backend/app/Http/Controllers/Api/v1/ClientsController.php

```diff
@@ -79,6 +79,8 @@ public function history(IndexClient $request): JsonResponse {
      * @return JsonResponse
      */
     public function store(StoreClient $request): JsonResponse {
+        Log::info('Ulozenie klienta do databazy: ', $request->only(['first_name', 'last_name']));
```

Comment:
Log davaj az potom co ta operacia aj realne zbehne, cize az za riadok ```Client::create($sanitized);```, to plati pre vsetko co si spravil, druha vec namiesto ```$request->only(['first_name', 'last_name']))``` pouzivaj ```$sanitized['first_name']```  ```$sanitized['last_name']```

Reply:
a este mi napadlo, ze ak to pises uz po slovensky tak aj diakritika